README.md

```diff
@@ -46,6 +46,11 @@ little more lenient. Here is the command I use in my personal i3wm config.
 bitwarden-dmenu --dmenu-args='-i' --clear-clipboard 30 --session-timeout 100 --sync-vault-after 3600 --on-error 'xargs notify-send --urgency=low'
 ```
 
+If you have the [password patch](https://tools.suckless.org/dmenu/patches/password/) for dmenu, you can run the following command to see your password as you enter it:
```

Comment:
hmm...I dont love this. I think its best to keep out of the readme. If you took the time to install the password patch, you probably already know to use it. As an alternative though, is it possible to programatically detect this? E.g. can we detect that the `-P` flag is available by calling `dmenu -h` and doing a text search? That would avoid the necessity for users to manually set things, and I doubt they would ever _not_ want this flag

Reply:
Text search of `dmenu -h` might not work for some people. For instance, if a patch fails and I have to manually add it, I just don't bother with editing the help message. This is the case in my current dmenu setup as well. We could just run `dmenu -P` and check whether that spits an error, but if the command has that option it will wait for user input before returning an exit code and that is not ideal either. We can just leave it as is right now and allow people to manually set it. If I ever find a solution though I will submit another pull request.